tests/test_mesh_reaction_rate_tally/test_mesh_reaction_rate_tally.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import hashlib
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import TestHarness
+from input_set import SimpleLatticeInput
+import openmoc.process as process
+import numpy as np
+
+
+class MeshReactionRateTallyTestHarness(TestHarness):
+    """An eigenvalue calculation with a mesh tally of the fission rates
+    using the openmoc.process module."""
+
+    def __init__(self):
+        super(MeshReactionRateTallyTestHarness, self).__init__()
+        self.input_set = SimpleLatticeInput()
+
+    def _run_openmoc(self):
+        """Run an OpenMOC eigenvalue calculation."""
+        super(MeshReactionRateTallyTestHarness, self)._run_openmoc()
+
+    def _get_results(self, num_iters=True, keff=True, fluxes=True,
+                     num_fsrs=True, num_tracks=True, num_segments=True,
+                     hash_output=True):
+        """Digest info from the mesh tallies and return as a string."""
+
+        # Create OpenMOC Mesh on which to tally fission rates
```

Comment:
Add the other options (cell/material, flux/total/scatter...) if tally_on_mesh is not tested elsewhere.

Reply:
Tests are ran in double precision, I doubt it's a simple rounding error. Let us know.

Reply:
Oh that's why. You need to compile your code in double precision, the default for installation is single precision